packages/fxa-settings/src/pages/Signin/index.tsx

```diff
@@ -127,7 +127,7 @@ const Signin = ({
           // TODO with signin_totp_code ticket, content server says this (double check it):
           // Login requests that ask for 2FA but don't have it setup on their account
           // will return an error.
-          navigate('/signin_totp_code');
+          navigate(`/signin_totp_code`);
```

Comment:
🤷 Why dis?

Reply:
I'm guessing Dan had passed location state or params here and then we didn't need them.

Related in my PR: https://github.com/mozilla/fxa/pull/16395/files#diff-890e7c3b34625d097298d21e96f001fe661164ee68054201ced4abffbe31b86eR142

Wondering if we want to take `verificationReason` out of the query params in this PR (unless I'm missing why we need it which could very well be the case) and then either leave it in local storage as Dan is doing it here or stick it in Apollo cache instead as I suggested over there 🤔

Reply:
It's now passed in reach router location state.